README.md

```diff
@@ -80,7 +80,7 @@ Allow support for repositories that use `git lfs` (default `false`).
 ### `ff_only`
 
 Refuse to merge and exit unless the current HEAD is already up to date or the
-merge can be resolved as a fast-forward (default `false`).
+merge can't be resolved as a fast-forward (default `false`).
```

Comment:
I'm pretty sure this change is still valid. Right? Or am I missing something here?

Reply:
It refuse the merge unless the merge can be resolved as fast forward, the current wording seems correct to me or I am missing something?